vividus/src/main/java/org/vividus/transformer/SortingTableTransformer.java

```diff
@@ -44,6 +44,9 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
         String byColumns = properties.getMandatoryNonBlankProperty("byColumns", String.class);
         String orderProperty = properties.getProperties().getProperty("order", "ASCENDING");
         Order order = (Order) fluentTrimmedEnumConverter.convertValue(orderProperty, Order.class);
+        String sortingTypeProperty = properties.getProperties().getProperty("sortingType", "STRING");
```

Comment:
how is it expected to work if we provide several columns for sorting?

Reply:
byColumns=key1|key2

Reply:
It works as now - sorting by key1 and if the same values by key2
Examples:
```

Scenario: Verify SORTING transformer with default order and NUMBER sortingType
When I initialize scenario variable `sortingTable` with values:
{transformer=SORTING, byColumns=key1|key2, sortingType = NUMBER}
|key1|key2|index|
|13  |100 |1    |
|13  |12  |2    |
|3   |10  |3    |
Then `${sortingTable}` is equal to table:
|key1|key2|index|
|3   |10  |3    |
|13  |12  |2    |
|13  |100 |1    |

Scenario: Verify SORTING transformer with default order and STRING sortingType
When I initialize scenario variable `sortingTable` with values:
{transformer=SORTING, byColumns=key1|key2, sortingType = string}
|key1|key2|index|
|13  |100 |1    |
|13  |12  |2    |
|3   |10  |3    |
Then `${sortingTable}` is equal to table:
|key1|key2|index|
|13  |100 |1    |
|13  |12  |2    |
|3   |10  |3    |
```

Reply:
I meant what if I want to sort key1 as number and key2 as string

Reply:
I think we need to give the ability to define different sorting types for different columns:
- `{transformer=SORTING, byColumns=key1|key2, sortingType = string|number}` - key1 is sorted as strings, key2 is sorted as numbers
- `{transformer=SORTING, byColumns=key1|key2, sortingType = number}` - all columns are sorted as numbers
- `{transformer=SORTING, byColumns=key1|key2}` - all columns are sorted as strings

Reply:
@valfirst could you please clarify what sortingType I should use for key3
`{transformer=SORTING, byColumns=key1|key2|key3, sortingType = string|number}`